src/qVoxFallProcess.cpp

```diff
@@ -439,7 +442,7 @@ bool qVoxFallProcess::Compute(const qVoxFallDialog& dlg, QString& errorMessage,
 	s_VoxFallParams.isEmptyBefore.resize(voxelGrid.innerCellCount(), true);
 	if (s_VoxFallParams.exportBlocksAsMeshes)
 	{
-		s_VoxFallParams.clusters.resize(voxelGrid.innerCellCount(), NULL);
+		s_VoxFallParams.clusters.resize(voxelGrid.innerCellCount(), 0);

```

Comment:
or nullptr? (haven't checked what the type is, sorry)

Reply:
it's a `std::vector<int> `so 0 should be fine.